src/main/java/pl/gov/coi/pomocua/ads/BaseOffer.java

```diff
@@ -59,6 +45,4 @@
     @LastModifiedDate
     public Instant modifiedDate;
 
-    @Transient
-    public String type = getClass().getSimpleName();
```

Comment:
And will the type of the entity be returned in `GET /api/secure/my-offers` endpoint in some other way?
Or the front-end doesn't need it?

Reply:
Nevermind, Hubert explained me how it works :) cool